### DISCUSSION TITLE
Purpose of `jsonSchema` extension?

### DISCUSSION THREAD

**Discussion by carlo-quinonez** [Urigo/graphql-scalars#2493] [Q&A] [answered]
We use JSONSchema extensively here at work. When I noticed that scalars included a [`jsonSchema` property](https://github.com/Urigo/graphql-scalars/blob/c85720aebadbb09c71fd023ee3299d4446a19e9b/src/scalars/json/JSONObject.ts#L18) under `extensions`, I was very curious what it does. I spent some time doing searches for `jsonSchema` and came up with no info whatsoever.

Can someone here may shed some light on how the `jsonSchema` extension is used?

**Comment by ardatan** [comment, marked as answer]
It is used by tools like SOFA that converts GraphQL Schema to JSON Schema or supersets of them (OpenAPI eg)
https://the-guild.dev/graphql/sofa-api

**Reply by carlo-quinonez** [reply]
Just curious... how did you know about this? There's only a single reference to `jsonSchema` in that code.

**Reply by ardatan** [reply]
I am one of the maintainers of both of them, and I implemented it 😅
